docs/changelog.md

```diff
@@ -1,3 +1,7 @@
+0.14.0.1
+
+- Null out the PYTHONPATH when executing any command to prevent conflicts between pipx dependencices and package dependenies when pipx is installed via homebrew. Homebrew can use pythonpath manipulation instead of virtual environments. (#233)
```

Comment:
Perhaps: `Remove the PYTHONPATH environment variable when...`?
Also, **dependencies**